### ISSUE TITLE
Improve SQL queries for index and shelve scripts

### ISSUE THREAD

**Issue by klassenjs** [PolarGeospatialCenter/pgcdemtools#82]
INFO- Fetching region lookup from Danco - takes about 5 minutes because it is (likely) reading the whole table (1,500,000 rows) into python memory for use later to join against generally a few thousand pairnames.  We can do much better.

**Comment by clairecporter** [comment]
Yes, we can use the [microservice](https://github.umn.edu/pgc/archive-resources-api) that Danny set up to get the region.  Although it will work better when Rory makes the changes he's working on to allow multiple items per API call.
